### DISCUSSION TITLE
Perfect square object

### DISCUSSION THREAD

**Discussion by ivanradojevic-web-dev** [tailwindlabs/tailwindcss#2455] [Help]
I insert w-64, h-64; How to achieve that proportions?

https://ibb.co/qBQGBp9

**Comment by mcmire** [comment]
Could you post some code?

**Comment by simonswiss** [comment]
Hey @ivanradojevic-web-dev, 

What do you mean by achieving that proportion? Do you want the brown shape in your screenshot to occupy the full space of the red square?
